go.mod

```diff
@@ -1,11 +1,14 @@
 module github.com/drone-plugins/drone-downstream
 
-go 1.14
+go 1.15
 
 require (
-	github.com/drone-plugins/drone-plugin-lib v0.3.1
-	github.com/drone/drone-go v1.3.1
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
+	github.com/drone-plugins/drone-plugin-lib v0.4.0
+	github.com/drone/drone-go v1.4.0
 	github.com/joho/godotenv v1.3.0
+	github.com/stretchr/testify v1.4.0
```

Comment:
@tboerger do you have any preferences on go testing modules? I don't have any strong opinions on it all.

Reply:
No strong opinion on that, but isn't the standard library enough?

Reply:
I can do without it, but it does make error checking in tests far less verbose.